pkg/cmd/cli/describe/printer.go

```diff
@@ -34,7 +34,7 @@ var (
 	imageColumns            = []string{"NAME", "DOCKER REF"}
 	imageStreamTagColumns   = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
 	imageStreamImageColumns = []string{"NAME", "DOCKER REF", "UPDATED", "IMAGENAME"}
-	imageStreamColumns      = []string{"NAME", "DOCKER REPO", "TAGS", "UPDATED"}
+	imageStreamColumns      = []string{"NAME", "TAGS", "UPDATED"}
```

Comment:
> hrm, "not every image can be docker pulled" - example? (I can think of online where the registry is not exposed with route). 

Example:
```
$ oc get is -n openshift
NAME         DOCKER REPO                               TAGS                          UPDATED
jenkins      172.30.208.39:5000/openshift/jenkins      1,latest                      3 minutes ago
mariadb      172.30.208.39:5000/openshift/mariadb      10.1,latest                   3 minutes ago
...
```
None of the above `DOCKER REPO` addresses is pullable. 

> just to double-check, we are exposing this information (docker pull spec) when you get the "oc get ist/foo:bar" right?

Correct, you're getting full docker pull spec including id of the image:
```
$ oc get istag/django-ex:latest
NAME               DOCKER REF                                                                                                  UPDATED              IMAGENAME
django-ex:latest   172.30.208.39:5000/test/django-ex@sha256:f39a7ee286d0c922c3838f0449f3364ee18d007ebfbcea6f9cefbceeb47b6933   About a minute ago   sha256:f39a7ee286d0c922c3838f0449f3364ee18d007ebfbcea6f9cefbceeb47b6933
```